apps/judicial-system/api/src/app/modules/case/dto/updateCase.input.ts

```diff
@@ -501,4 +501,9 @@ export class UpdateCaseInput {
   @IsOptional()
   @Field(() => CourtSessionType, { nullable: true })
   readonly courtSessionType?: CourtSessionType
+
+  @Allow()
+  @IsOptional()
+  @Field(() => ID, { nullable: true })
+  readonly mergeCaseId?: string
```

Comment:
Please note that we should guard against updating this field being updated if the case is not in state `RECEIVED`. This can be done in the case controller in the method `update` by throwing a `BadRequestException` it the state is not valid.

Also note that when implementing the transition that completes the case, that can be done in the case service in the `update` method. This is already being done when receiving cases.

Reply:
True, I'll make the change